android/source/VideoLocker/src/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -764,6 +763,13 @@ private boolean showErrorMessage(String header, String message) {
     }
 
     /**
+     * This method should be implemented by {@link org.edx.mobile.view.MyCoursesListActivity}.
+     */
+    protected void reloadMyCoursesData() {
```

Comment:
Having logic specific to *one* activity in the class *all* activities subclass from seems weird to me. Maybe we should better configure how we handle these messages.